vendors/maddalena/models/arrowwan2/model.yaml

```diff
@@ -0,0 +1,46 @@
+# Commercial name of the model 
+name: ArrowWan 2
+# Functional description of the product. Maximum 500 characters. 
+description: "Compact wMBus and LoRaWAN™ radio module, ensuring a high level of interoperability with a range of mobile and fixed network reading systems, including third-party ones. Arrow WAN 2 is also suitable for difficult Utility installations (IP68). Activation and configuration of the multiprotocol radio are carried out locally using the NFC interface and a dedicated Android app."
+# Logo of the device
+logo: arrowwan2.png
+# Product name & Version
+product:
+    name: MADDALENA/ARROWWAN2
+    version: 1
```

Comment:
```suggestion
```

No need for this part in the new device models. It was used before in the already existing devices.